src/relay/transforms/partition_graph.cc

```diff
@@ -480,11 +480,77 @@ IRModule FlattenTupleOutputs(IRModule module) {
   return module;
 }
 
+class Mangler : public MixedModeMutator {
```

Comment:
this one seems pretty easy to unit test--possible to write one?

Reply:
The point is that now the pass is integrated inside `the relay._transform.PartitionGraph` pass, so there is now way to test this in isolation. This will be tested by every BYOC test